src/main/java/com/spotify/reaper/service/SegmentRunner.java

```diff
@@ -152,7 +156,20 @@ private void runRepair() {
       RepairUnit repairUnit = context.storage.getRepairUnit(segment.getRepairUnitId()).get();
       String keyspace = repairUnit.getKeyspaceName();
 
-      if (!canRepair(segment, keyspace, coordinator)) {
+      LazyInitializer<Set<String>> busyHosts = new LazyInitializer<Set<String>>() {
```

Comment:
Add a comment explaining why it's a `LazyInitializer`.  I think I get why, but it took reading some code to figure it out.